app/components/Twitter.tsx

```diff
@@ -0,0 +1,210 @@
+import Vue from 'vue';
+import TsxComponent from 'components/tsx-component';
+import { Component, Prop, Watch } from 'vue-property-decorator';
+import { Inject } from 'services/core/injector';
+import { TwitterService } from 'services/integrations/twitter';
+import HFormGroup from 'components/shared/inputs/HFormGroup.vue';
+import { $t } from 'services/i18n';
+import { shell } from 'electron';
+import { Spinner, TextArea, Button } from 'streamlabs-beaker';
+import { ToggleInput } from 'components/shared/inputs/inputs';
+import { UserService } from 'services/user';
+
+import cx from 'classnames';
+import styles from './Twitter.m.less';
+
+@Component({})
+export class Twitter extends TsxComponent<{
+  streamTtitle: string;
+  midStreamMode: boolean;
+  updatingInfo: boolean;
+  value: string;
+}> {
+  @Inject() twitterService: TwitterService;
+  @Inject() userService: UserService;
+
+  @Prop() streamTitle: string;
+  @Prop() midStreamMode: boolean;
+  @Prop() updatingInfo: boolean;
```

Comment:
looks like you're missing the `value` prop here, unless that's unused in which case you can remove it from the type declaration above

Reply:
Got rid of tweetModel and used `value` prop where I need it